test_conformance/api/test_api_min_max.cpp

```diff
@@ -22,16 +22,16 @@
 const char *sample_single_param_kernel[] = {
     "__kernel void sample_test(__global int *src)\n"
     "{\n"
-    "    int  tid = get_global_id(0);\n"
+    "    size_t  tid = get_global_id(0);\n"
     "\n"
     "}\n"
 };
 
 const char *sample_single_param_write_kernel[] = {
     "__kernel void sample_test(__global int *src)\n"
     "{\n"
-    "    int  tid = get_global_id(0);\n"
-    "     src[tid] = tid;\n"
+    "    size_t  tid = get_global_id(0);\n"
+    "    src[tid] = tid;\n"
```

Comment:
There's still a potential issue assigning `size_t` to `int` here. We should make that cast explicit, and determine if the test is affected by the overflow situation.

Reply:
This kernel is unused, so I'm just removing it.